homeassistant/components/mpd/media_player.py

```diff
@@ -251,6 +255,54 @@ def media_album_name(self):
         """Return the album of current playing media (Music track only)."""
         return self._currentsong.get("album")
 
+    @property
+    def media_image_hash(self):
+        """Hash value for media image."""
+        file = self._currentsong.get("file")
+        if file:
+            return hashlib.sha256(file.encode("utf-8")).hexdigest()[:16]
+
+        return None
+
+    async def async_get_media_image(self):
```

Comment:
Please don't add new features in this PR. Just do the async refactor.

Reply:
Alright, postponing this until the async changes are in.